File: app/src/main/java/com/intelegain/agora/model/ChangePassword.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ChangePassword {

    @SerializedName("Status")
    @Expose
    public String status;
    @SerializedName("Message")
    @Expose
    public String message;
}

File: app/src/main/java/com/intelegain/agora/model/ChangePassword.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ChangePassword {
    @kotlin.jvm.JvmField
    @SerializedName("Status")
    @Expose
    var status: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("Message")
    @Expose
    var message: String? = null
}